src/plugins/run/code.ts

```diff
@@ -156,6 +156,7 @@ function handleLegacyAnnotations(annotations: MDAST.Code['annotations']) {
     // @ts-expect-error type is no longer part of the typings, it is deprecated
     const type = annotations.type;
     if (type === 'json_schema') {
+      // @ts-expect-error
```

Comment:
Can't we just change the value of `jsonSchema` to true?

Reply:
Yeah but if we do that then the output in the HTML is `jsonSchema`, which ultimately translates to `jsonSchema=""`. Per above comment this just seemed a bit confusing to me (versus explicit "true"/"false" values), but I can change it back if you feel strongly about it no problem just LMK.

<img width="660" alt="Screen Shot 2021-06-17 at 12 57 03 PM" src="https://user-images.githubusercontent.com/847542/122441412-24244200-cf63-11eb-99cc-45d6048f4694.png">